spec/helpers/item_seasons_helper_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ItemSeasonsHelper, type: :helper do
+  describe 'rotation' do
+    it 'adds rotation to month with index 12' do
+      expect(helper.rotation(12)).to eq('rotate-plus-10')
+    end
```

Comment:
**issue**: We should cover all cases, here you are just verifying that `12` yields the expected output. If we refactored the method to
```rb
def rotation(index)
  return 'rotate-plus-10' if index == 12
  
  'rotate-zero'
end
```

Tests would still pass.

Reply:
I added tests for all the rotations